tests/security/selinux/selinux_setup.pm

```diff
@@ -68,8 +69,15 @@ sub run {
     }
 
     # record the pkgs' version for reference
-    my $results = script_output("zypper se -s selinux-policy");
-    record_info("Pkg_ver", "SELinux policy packages' version is: $results");
+    my $results = script_output("zypper se -s selinux policy");
+    record_info("Pkg_ver", "SELinux packages' version is: $results");
```

Comment:
Seem so many lines printing out there, and you record it as package version, can you please do some enhancements here?
anyway I don't have good suggestion, just some wondering.

Reply:
I want to check all related packages's version so I prefer this record info style.

Reply:
Then no issue